src/fides/core/export.py

```diff
@@ -148,6 +148,21 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
     formatted_data_subjects, custom_data_subjects = format_data_subjects(
         server_resources["data_subject"]
     )
+
+    privacy_declaration_fields = [
```

Comment:
There used to be a magic variable below that was the number 12. This converts it to a be a dynamic variable